pages/clubs/Running.jsx

```diff
@@ -41,7 +41,7 @@ var Running = React.createClass({
     httpRequest.send();
   },
   componentDidMount: function() {
-    this.fetchJSON("https://mozilla.github.io/learning-networks/clubs/clubs-resources.json",this.setData);
+    this.fetchJSON("https://teach-api-staging.herokuapp.com/api/clubsguides/categories?expand=clubsguides",this.setData);
```

Comment:
Ah neat! I'll fix that when I'm actually going to merge this in since for now it's just for QA